src/libraries/System.ComponentModel.TypeConverter/src/System/Timers/Timer.cs

```diff
@@ -62,6 +65,9 @@ public Timer(double interval) : this()
         /// <summary>
         /// Initializes a new instance of the <see cref='Timer'/> class, setting the <see cref='Interval'/> property to the specified period.
         /// </summary>
+        /// <param name="interval">
+        /// The time between events. The value in milliseconds must be greater than zero and less than or equal to <see cref="int.MaxValue"/>.
```

Comment:
This one is a `TimeSpan`. So what should be the max allowed value for it?

Reply:
Ah nevermind yes. You are already explaining that the _milliseconds_ are not to be larger than int.MaxValue.